tests/conftest.py

```diff
@@ -25,21 +25,14 @@
 RUN_PRED = "tests/data/drogon/ertrun1/realization-0/pred"
 
 
-def pytest_configure():
```

Comment:
Think this is fine to remove now, as we patch the `detect_inside_rms(...)` function during testing (if need be).